File: ManagementPermissionAPI/ManagementPermission.Application/Dto/Permission/PermissionDto.cs
using ManagementPermission.Application.Common;
using ManagementPermission.Application.Dto.TypeOfPermission;
using System;

namespace ManagementPermission.Application.Dto.Permission
{
    public class PermissionDto : BaseEntityDto
    {
        public string Name { get; set; }
        public string LastName { get; set; }
        public DateTime Date { get; set; }
        public int TypeOfPermissionId { get; set; }
        public TypeOfPermissionDto TypeOfPermission { get; set; }
    }
}

File: permissionsapp/src/models/PermissionDto.ts
import { TypeOfPermissionDto } from "./TypeOfPermissionDto";

export interface PermissionDto {
  name?: string;
  lastName?: string;
  type?: number;
  typeOfPermissionId?: number;
  typeOfPermission?: TypeOfPermissionDto;
  id?: number;
  created?: Date;
}
